src/Microsoft.ML.StandardLearners/Standard/Simple/SimpleTrainers.cs

```diff
@@ -38,20 +38,16 @@ namespace Microsoft.ML.Runtime.Learners
     /// <summary>
     /// A trainer that trains a predictor that returns random values
     /// </summary>
-    public sealed class RandomTrainer : TrainerBase<RandomPredictor>
+
+    public sealed class RandomTrainer : TrainerBase<RandomPredictor>,
```

Comment:
>TrainerBase<RandomPredictor> [](start = 40, length = 28)

I think this can go away too. It will eventually, to my understanding. @Zruty0 can confirm.

Reply:
Yes, this is an exception :)

---
In reply to: [218156771](https://github.com/dotnet/machinelearning/pull/876#discussion_r218156771) [](ancestors = 218156771,218152453)